frontera/settings/__init__.py

```diff
@@ -38,7 +56,7 @@ def __setattr__(self, name, value):
             self.__dict__[name] = value
 
     def add_module(self, module):
-        if isinstance(module, Settings):
+        if isinstance(module, BaseSettings):
```

Comment:
Yes, since the beginning I felt this behavior weird. IMO we should be more explicit, like having a ```from_module(str)``` for those who want a new Settings instance and the rest of the code should only work with Settings objects.